src/Tiamat_oxDNA.py

```diff
@@ -197,6 +198,54 @@ def get_pos(self):
     def get_across(self):
         return self  # Returns itself to reduce memory consumtion 
 
+def getBaseConfig(base):
```

Comment:
More nitpicking: we use CamelCase for class names and underscore_separated_names for method and function names. Can you change this to `get_base_config`?

Reply:
No worries, I should have noticed that. Fixed now :)